### PR TITLE
Press F to pay respects

### PR DESCRIPTION
**Pull request by katlings**
After a player dies, other players can press `f` to display their respects in the killfeed.

* This does overlap with the flashlight key for hunters - `f` will toggle both respects and the flashlight ¯\\\_(ツ)_/¯
* Includes the fake death spawn fix from #13

### CODE REVIEW

**Comment by kvovin** [comment]
Nice.

**Comment by Calvin-L** [comment]
> `f` will toggle both respects and the flashlight

That's a feature

tenor

File: gamemode/client/hotkeys.lua [hunk 1]
```diff
@@ -33,6 +33,16 @@ hook.Add("KeyPress", "PressShiftHunterHintUpdown", function(ply, key)
...
 
+hook.Add("PlayerButtonDown", "PressFToPayRespects", function(ply, button)
+    if (CLIENT and
+        IsFirstTimePredicted() and
+        button == KEY_F
+    ) then
+        net.Start("Pay Respects")
+        net.SendToServer()
+    end
+end)
```
Comment (Calvin-L) [inline gamemode/client/hotkeys.lua:44]: Can this be a server-side hook? A client-side hook that sends the server a message screams "server-side hook" to me.

Comment (katlings) [reply gamemode/client/hotkeys.lua:44]: whoops lol I just assumed a server wouldn't have a hook for a client event

yeah that def sounds like the right thing, but I'm not going to have a chance to change it before tonight 😔 feel free to upload an edit or I can clean it up later

Comment (katlings) [reply gamemode/client/hotkeys.lua:44]: this despite our discussion that would have made it clear if I'd thought about it for two seconds...

Comment (kvovin) [reply gamemode/client/hotkeys.lua:44]: I modelled this poorly, I didn't realized some of these hooks worked in the SERVER context as well. Why don't we merge as is and we'll do a cleanup later.

**Comment by Calvin-L** [comment]
Merged! We'll clean it up later.

For sure.

Definitely.

😆

**Pull request merged**